pkgs/top-level/all-packages.nix

```
@@ -8054,15 +8054,15 @@ with pkgs;
 
   phosh = callPackage ../applications/window-managers/phosh { };
 
-  pinentry = libsForQt5.callPackage ../tools/security/pinentry {
+  pinentryFlavors = callPackages ../tools/security/pinentry {
     libcap = if stdenv.isDarwin then null else libcap;
   };
```

```suggestion
  pinentryFlavors = lib.recurseIntoAttrs (callPackages ../tools/security/pinentry {
    libcap = if stdenv.isDarwin then null else libcap;
  });
```

To make them show on hydra.